routemodel/data_retrieval/elevations.py

```diff
@@ -18,7 +16,7 @@
                 57: "5", 58: "6", 59: "7", 60: "8", 61: "9", 62: "_", 63: "-"}
 
 
-def points_builder(coordinates: list):
+def elevations_points_builder(coordinates: list):
```

Comment:
Nitpicky - I would favour `build_x` instead of `x_builder` since it matches the other method names